ee/tabby-webserver/src/handler.rs

```diff
@@ -8,7 +8,10 @@ use axum::{
 };
 use hyper::Body;
```

Comment:
I was also wondering why `handler.rs` duplicates the code from `lib.rs`, why not keep only one copy?

Reply:
it's a mistake - we shall only keep the content inside `handler.rs`